bin/PrintGCStats

```diff
@@ -805,6 +807,19 @@ function writePlotData(tstamp, value, widx) {
 }
 
 function recordStats(idx, value) {
+  rs_tstamp = -1;
+  if (available_plot_count != 0 && (idx in plot_idx || (totgc_idx in plot_idx && idx < totgc_idx))) {
+    # Plotting enabled, need to find timestamp on line
+    if (datestamps == 1) {
+      rs_tstamp = getDateTimeStamp();
+    } else {
+      rs_tstamp = getTimeStamp();
+    }
+  }
+  recordStatsTimestamp(idx, value, rs_tstamp);
```

Comment:
Hi

I am away from my computer for a couple of weeks, I will look into this and update my pull requests when I am back.

Alf

Sendt fra min iPad

> Den 19. feb. 2015 kl. 00.08 skrev joyxiong <notifications@github.com>:
> 
> In bin/PrintGCStats:
> 
> > @@ -805,6 +807,19 @@ function writePlotData(tstamp, value, widx) {
> >  }
> >  
> >  function recordStats(idx, value) {
> > +  rs_tstamp = -1;
> > +  if (available_plot_count != 0 && (idx in plot_idx || (totgc_idx in plot_idx && idx < totgc_idx))) {
> > +    # Plotting enabled, need to find timestamp on line
> > +    if (datestamps == 1) {
> > +      rs_tstamp = getDateTimeStamp();
> > +    } else {
> > +      rs_tstamp = getTimeStamp();
> > +    }
> > +  }
> > +  recordStatsTimestamp(idx, value, rs_tstamp);
> Can you clarify why you restructure the code into recordSrarsTimestamp and recordStats()? Are you trying to cover a specific log record by having recordStatsTimestamp?
> 
> —
> Reply to this email directly or view it on GitHub.
>